asset/css/site-nav.css

```diff
@@ -8,8 +8,9 @@
 #page-content {
     flex: 1;
     margin-left: 285px;
+    overflow-x: hidden;
     padding-left: 20px;
```

Comment:
Then the page contents will touch the divider line between itself and the site navigation. 

It is not obvious in our documentation since `website-content` class has additional margins to space things out. 

I removed `padding-right` as there is a built in padding from the app container already which helps us avoid being too close to the scroll bar.